repose-aggregator/components/filters/header-user-filter/pom.xml

```diff
@@ -48,6 +48,7 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.MajDotMin}</artifactId>
+            <scope>test</scope>
```

Comment:
My thought is that it won't be around long enough to matter one way or the other.